query-model/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/impl/JpqlMod.kt

```diff
@@ -0,0 +1,13 @@
+package com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl
+
+import com.linecorp.kotlinjdsl.Internal
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+
+/**
+ * Expression that calculates [value1] (mod [value2]).
```

Comment:
I noticed that there are parentheses before and after the mod, what is the intent?